centreon/tests/e2e/features/Dashboards/01-dashboard-creation/index.ts

```diff
@@ -213,7 +213,7 @@ When('the dashboard administrator user starts to edit the dashboard', () => {
 });
 
 Then("creates a new dashboard on the previous dashboard's edition page", () => {
-  cy.getByTestId({ testId: 'ArrowDropDownIcon' }).click();
+  cy.getByTestId({ testId: 'ArrowDropDownIcon' }).click({multiple: true});
```

Comment:
why clicking on multiple arrows ?
e2e test should reflect what the used does
You should identify which arrow you need to click (aria-label, id ...) or wait until only one is visible